src/views/ModelPlan/TaskList/ParticipantsAndProviders/_component/ProviderAndSupplierTable/index.tsx

```diff
@@ -0,0 +1,160 @@
+  const {
+    getTableProps,
+    getTableBodyProps,
+    gotoPage,
+    headerGroups,
+    nextPage,
+    page,
+    pageOptions,
+    previousPage,
+    canNextPage,
+    canPreviousPage,
+    pageCount,
+    setPageSize,
+    setGlobalFilter,
+    state,
+    rows,
+    prepareRow
+  } = useTable(
+    {
+      columns,
+      data,
+      initialState: { pageIndex: 0 }
+    },
+    useGlobalFilter,
+    useSortBy,
+    usePagination
+  );
+
+  // Render the UI for your table
+  return (
+    <>
+      <GlobalClientFilter
+        globalFilter={state.globalFilter}
+        setGlobalFilter={setGlobalFilter}
+        tableID={modalT('modal.table.id')}
+        tableName={modalT('modal.table.title')}
+        className="margin-bottom-4 maxw-none width-mobile"
+      />
+      <TableResults
+        globalFilter={state.globalFilter}
+        pageIndex={state.pageIndex}
+        pageSize={state.pageSize}
+        filteredRowLength={rows.length}
+        rowLength={data.length}
+      />
+      <UswdsTable bordered={false} {...getTableProps()} fullWidth>
+        <caption className="usa-sr-only">
+          {modalT('modal.table.caption')}
+        </caption>
+        <thead>
+          {headerGroups.map(headerGroup => (
+            <tr {...headerGroup.getHeaderGroupProps()}>
+              {headerGroup.headers.map(column => (
+                <th
+                  {...column.getHeaderProps()}
+                  className={`border-width-2px padding-bottom-1 ${
+                    column.id === 'providerType' ? 'width-card' : ''
+                  }`}
+                  scope="col"
+                >
+                  <button
+                    className="usa-button usa-button--unstyled position-relative"
+                    type="button"
+                    {...column.getSortByToggleProps()}
+                  >
+                    {column.render('Header')}
+                    {getHeaderSortIcon(column, false)}
+                  </button>
+                </th>
+              ))}
+            </tr>
+          ))}
+        </thead>
+        <tbody {...getTableBodyProps()}>
+          {page.map((row, i) => {
+            prepareRow(row);
+            return (
+              <tr {...row.getRowProps()}>
+                {row.cells.map(cell => {
+                  return (
+                    <td {...cell.getCellProps()}>{cell.render('Cell')}</td>
+                  );
+                })}
+              </tr>
+            );
+          })}
+        </tbody>
+      </UswdsTable>
+      <TablePagination
+        className="flex-justify-start"
+        gotoPage={gotoPage}
+        previousPage={previousPage}
+        nextPage={nextPage}
+        canNextPage={canNextPage}
+        pageIndex={state.pageIndex}
+        pageOptions={pageOptions}
+        canPreviousPage={canPreviousPage}
+        pageCount={pageCount}
+        pageSize={state.pageSize}
+        setPageSize={setPageSize}
+        page={[]}
+      />
+    </>
```

Comment:
I have a ticket to capture all the conversions of table pagination [MINT-3034](https://jiraent.cms.gov/browse/MINT-3034).  With the latest upgrade to Truss package we now have access to their pagination.  In my last PR I created a hook to merge the component with functionality.  We should standardize all pagination now.  This won't block the PR but just wanted to note this for future visibility.  The mentioned ticket will rewrite this when worked on.  The hook is called `usePagination` and can be referenced by the Models Approaching Clearance PR that was jsut merged